docs/v3_services.rst

```diff
@@ -0,0 +1,249 @@
+Securedrop V3 Onion Services
+============================
+
+SecureDrop uses onion services for the *Journalist* and *Source Interfaces*, as
+well as for SSH access in SSH-over-Tor mode. Previous to the
+release of SecureDrop 1.0.0, only v2 onion services were available, but 
+administrators now have the option of enabling next-generation (v3) onion 
+services.
+
+V3 onion services use more secure cryptography than v2 onion services,  and 
+include redesigned protocols that guard against service 
+information leaks on the Tor network. The unique identifier in V3 onion 
+addresses is 56 characters long - for example:
+
+.. code-block:: none
+
+  http://vww6ybal4bd7szmgncyruucpgfkqahzddi37ktceo3ah7ngmcopnpyyd.onion
+
+This makes them easily distinguishable from 16-character v2 onion addresses,
+such as:
+
+.. code-block:: none
+
+  https://www.nytimes3xbfgragh.onion/
```

Comment:
Tiny nit: one example URL has a trailing slash, the other doesn't. It might also be good to have them use the same protocol (HTTPS or HTTP) so that the reader doesn't associate that difference with v2 vs. v3.

Reply:
fixed trailing slash, swapped out the HTTPS onion for a HTTP one.